.travis.yml

```diff
@@ -7,7 +7,7 @@ install:
     - pip install -r requirements-optional.txt coveralls
 script:
     - flake8 .
-    - py.test --cov=automation_tools tests/
+    - py.test --cov=automation_tools tests/test_log.py
```

Comment:
Keeping as it is will also runs post upgrade verification tests, which needs preupgrade.pkcl and postupgrade.pkcl files. Which can only be available by running set_datastore with fabric.

Also we are planning to separate out whole upgrade in separate project, so this is just a POC.